cmd/dashboard/console.js

```diff
@@ -38,7 +39,13 @@ class Console {
     var pluginCmds = this.plugins.getPluginsProperty('console', 'console');
     for (let pluginCmd of pluginCmds) {
       let pluginResult = pluginCmd.call(this, cmd, {});
-      if (pluginResult.match()) {
+      if(typeof pluginResult !== 'object'){
+        if (pluginResult !== false && pluginResult !== 'false' && pluginResult !== undefined) {
+          this.logger.warn("[DEPRECATED] In future versions of embark, we expect the console command to return an object " +
+            "having 2 functions: match and process.");
```

Comment:
We could put a link to the doc here (https://embark.status.im/docs/plugin_reference.html#embark-registerConsoleCommand-callback-options) (we will obviously need to update the doc)